adit/dicom_web/views.py

```diff
@@ -363,3 +413,17 @@ async def post(
 
         assert request.accepted_renderer is not None
         return Response([results], content_type=request.accepted_renderer.media_type)
+
+
+def is_valid_pseudonym(pseudonym: str) -> None:
```

Comment:
Not a good fit here. Move it over to validators. It just a validator that uses other validators.